content/article/saying-goodbye-to-search-cpan-org.md

```diff
@@ -0,0 +1,87 @@
+
+  {
+    "title"       : "Saying Goodbye to search.cpan.org",
+    "authors"     : ["olaf-alders"],
+    "date"        : "2018-06-26T09:50:46",
+    "tags"        : ["CPAN", "MetaCPAN"],
+    "draft"       : true,
+    "image"       : "",
+    "thumbnail"   : "",
+    "description" : "",
+    "categories"  : "development"
+  }
+
+If you've visited [search.cpan.org](http://search.cpan.org) in the last day or
+so, you may have noticed that this CPAN search site is now directing all of its
+traffic to [metacpan.org](https://metacpan.org).  Let's talk about what this change
+means.
+
+## Why is this change taking place?
+
+The maintainers behind [search.cpan.org](http://search.cpan.org) have decided
+that it's time to move on.  After many, many years of keeping this site up and
+running they have decided to pass on the torch (and the traffic) to
+[metacpan.org](https://metacpan.org).  The MetaCPAN team has been working very
+hard to prepare for the influx of new traffic (and new users).  On behalf of
+Perl users everywhere, the MetaCPAN team would like to thank all of the crew
```

Comment:
Do you want to mention their names (if you know them)?

Reply:
I did, but I'm not sure I know *all* of them.  So, I could thank a possibly incomplete list or thank them all with a generic blanket statement.  Do you have a preference?  I can follow up with them to get a proper list.  (Sorry, just saw this now).

Reply:
Gotcha, as this is time sensitive let's not wait for the full list and stick with the blanket statement. Just let me know what you think about PR #111 and we'll publish